frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/atoms/UploadServices.js

```
@@ -22,9 +22,9 @@ export const UploadServices = {
     const formData = new FormData();
     const filesArray = Array.from(filesData)
     filesArray?.forEach((fileData, index) => fileData ? formData.append("file", fileData, fileData.name) : null);
-    formData.append("tenantId", tenantId);
+    // formData.append("tenantId", tenantId);
     formData.append("module", module);
-    let tenantInfo=window?.globalConfigs?.getConfig("ENABLE_SINGLEINSTANCE")?`?tenantId=${tenantId}`:"";
+    let tenantInfo=`?tenantId=${tenantId}`;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'tenantInfo' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)